doc/sphinx/source/references.bib

```diff
@@ -134,3 +134,28 @@ @article{zhang2011verification
   year={2011},
   pages={807--827}
 }
+
+@book{bower2010applied,
+  title={Applied mechanics of solids},
+  author={Bower, Allan F},
+  year={2010},
+  publisher={CRC press},
+  isbn={978-1-4398-0247-2}
+}
+
+
+@book{Holzapfel2002,
+  title     = "Nonlinear Solid Mechanics: A Continuum Approach for Engineering
+               Science",
+  author    = "Holzapfel, Gerhard A",
+  abstract  = "Introduction to Vectors and Tensors. Kinematics. The Concept of
+               Stress. Balance Principles. Some Aspects of Objectivity.
+               Hyperelastic Materials. Thermodynamics of Materials. Variational
+               Principles. References. Index.",
+  journal   = "Meccanica",
+  publisher = "Springer Nature",
+  volume    =  37,
+  number    = "4/5",
+  pages     = "489--490",
```

Comment:
This looks like a mangled citation, perhaps for a review of the book rather than for the book itself.